src/main/resources/theme/style.css

```diff
@@ -711,10 +711,7 @@
 /***************** Main window *****************/
 
 .main-window {
-  -fx-background-image: url('images/login-background.jpg');
-  -fx-background-repeat: no-repeat;
-  -fx-background-position: center;
-  -fx-background-size: cover, auto;
+
 }
```

Comment:
Was hesitating if I should have removed the entire class, as well as from styleClass="main-window" in the fxml. I guess I should?